images/router/haproxy/reload-haproxy

```diff
@@ -1,21 +1,98 @@
 #!/bin/bash -xu
```

Comment:
It was there from before and I sort like it (it helps with debugging - tells you what the script did/is doing).